nucleus/common/simple-glassfish-api/src/main/java/org/glassfish/embeddable/GlassFishProperties.java

```diff
@@ -74,8 +76,10 @@ public GlassFishProperties() {
      * for the plugged in {@link GlassFishRuntime} (if any)
      *
      * @param props Properties object which will back this GlassFishProperties object.
+     * @throws NullPointerException if props is null
      */
     public GlassFishProperties(Properties props) {
+        Objects.requireNonNull(props);
```

Comment:
With an error message it would be beautiful ... I know this NPE thrown a bit later for years! :smiling_imp: 
Btw copyright 2017 - is it correct?

Reply:
I think there is a default error message that's there even if not specified

Reply:
I was wrong :)

Reply:
Since JDK 14 https://openjdk.java.net/jeps/358